tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -252,6 +252,9 @@ def _switch_transpose_and_node(self, node, trans):
     # if return value is True, then it means Transpose is handled as designed
     # otherwise, it means that we skip handling since it is not in our support set
     def _handle_nhwc_tranpose(self, trans):
+        if trans.output[0] in self._g.outputs:
+            log.debug("%s connects to graph outputs", trans.output[0])
+            return False

```

Comment:
add a test to cover this case?

Reply:
need f850cbe15a3a3e928274e8d94400d756846baf7f

Reply:
yep

Reply:
@lucienwang1009, a simple case will be enough, tks.